packages/api/src/modules/user/user.service.ts

```diff
@@ -176,7 +176,7 @@ export class UserService {
         await this.validateEmail(email.toLocaleLowerCase());
 
         if (await userRepository.findByEmail(email.toLocaleLowerCase()))
-            throw new ConflictError("User is already exist", UserServiceErrors.CREATE_USER_ALREADY_EXIST);
+            throw new ConflictError("User already exist", UserServiceErrors.CREATE_USER_ALREADY_EXIST);
```

Comment:
```suggestion
            throw new ConflictError("User already exists", UserServiceErrors.CREATE_USER_ALREADY_EXIST);
```

on peut renomme l'erreur dans l'enum aussi